webview-ui/src/components/settings/ApiOptions.tsx

```diff
@@ -85,7 +85,13 @@ const SUPPORTED_THINKING_MODELS: Record<string, string[]> = {
 		"qwen-plus-latest",
 		"qwen-turbo-latest",
 	],
-	gemini: ["gemini-2.5-flash-preview-05-20", "gemini-2.5-flash-preview-04-17", "gemini-2.5-pro-preview-06-05"],
+	gemini: [
+		"gemini-2.5-flash-preview-05-20",
+		"gemini-2.5-flash-preview-04-17",
+		"gemini-2.5-pro-preview-06-05",
+		"gemini-2.5-pro",
```

Comment:
<img width="254" alt="image" src="https://github.com/user-attachments/assets/7fb479d9-41da-4e85-8678-efd43190b9f6" />


Get rid of these 3 we don't support them 

<img width="399" alt="image" src="https://github.com/user-attachments/assets/8e0909b3-c73f-4388-a45d-d01787d7c15f" />


as they have been condensed